### ISSUE TITLE
Loss value for successful runs

### ISSUE THREAD

**Issue by dinarkino** [hustvl/YOLOP#122]
If anyone successfully trained the model for specific tasks or for all tasks, could you please share loss values or loss graphs? Also, number of epochs and hyperparameters could be helpful. It will be easier to understand if there is something wrong during training and how to obtain the best results. Or maybe the authors of the repo could provide such information?
